File: csharp/line-up/LineUp.cs
public static class LineUp
{
    public static string Format(string name, int number)
    {
        int lastDigit = number % 10;
        int lastTwoDigit = number % 100;
        string suffix;
        if (lastDigit == 1 && !(lastTwoDigit == 11))
            suffix = "st";
        else if (lastDigit == 2 && !(lastTwoDigit == 12))
            suffix = "nd";
        else if (lastDigit == 3 && !(lastTwoDigit == 13))
            suffix = "rd";
        else
            suffix = "th";

        return $"{name}, you are the {number}{suffix} customer we serve today. Thank you!";
    }
}

File: javascript/line-up/line-up.js
//
// This is only a SKELETON file for the 'Line Up' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const format = (name, pos) => {
  let lastDigit = pos % 10;;
  let lastTwoDigits = pos % 100;
  let suffix = "";

  if (lastDigit == 1 && !(lastTwoDigits == 11))
    suffix = "st";
  else if (lastDigit == 2 && !(lastTwoDigits == 12))
    suffix = "nd";
  else if (lastDigit == 3 && !(lastTwoDigits == 13))
    suffix = "rd";
  else
    suffix = "th";

  return `${name}, you are the ${pos}${suffix} customer we serve today. Thank you!`
};
